doc/how_to/observers/ipywidget.md

```diff
@@ -0,0 +1,186 @@
+# Observe ipywidgets
```

Comment:
Should we move the Traitlets functionality to Param @philippjfr ? I would like to avoid the effort. So it should be important.

Reply:
Sounds intriguing.